api/src/index.js

```diff
@@ -11,14 +11,70 @@ app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+// Getting future meals
+apiRouter.get("/future-meals", async (req, res) => {
+	try {
+		const upcomingMeals = await knex.raw(
+			"SELECT * FROM meal WHERE `when` > NOW()"
+		);
+		const meals = upcomingMeals[0];
+		meals.length ?
+			res.json(meals)
+		:	res.status(404).json("There are no any events in the future.");
```

Comment:
For routes that are supposed to return multiple elements, (`all-meals`, `future-meals` etc.) you should not return `404` error, as an empty array is a valid response: it lets the user know that there are no meals. Consider the case where Facebook fetches messages: if you have not sent a message to a person yet, you should not see an error, but an empty chat.